<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="Transcription"/>
    <title>Transcription</title>
    <link rel="stylesheet" type="text/css" href="../../style.css" media="screen">
</head>

<body>

<nav>
	<ul>
		<li><a class="navbar" href="../index.html">Home</a></li>
		<li><a class="navbar" href="index.html">Index</a></li>
	</ul>
</nav>

<div class="content">
 
<h1>Transcription</h1>

<p>A <b>gene</b> may be defined as a sequence of nucleotides that is necessary and sufficient for the synthesis of a polypeptide or a stable RNA molecule such as a ribosomal or transfer RNA (rRNA or tRNA). For example, enzymes, including those that metabolize nutrients and synthesize new cellular constituents, as well as DNA polymerases and other enzymes that make copies of DNA during cell division, are all proteins.</p>

<pre>DNA ⟶ RNA</pre>

<p>In the simplest sense, expressing a gene means manufacturing its corresponding protein, and this multilayered process has two major steps. In the first step, the information in DNA is transferred to a messenger RNA (mRNA) molecule by way of a process called transcription. During transcription, the DNA of a gene serves as a template for complementary base-pairing, and an enzyme called <b>RNA polymerase II</b> catalyzes the formation of a pre-mRNA molecule, which is then processed to form mature mRNA (Figure 1). The resulting mRNA is a single-stranded copy of the gene, which next must be translated into a protein molecule. </p>

<p>In eukaryotes, mRNA precursors (pre-mRNAs) are transcribed by RNA polymerase II (Pol II) from the genome and must undergo extensive cotranscriptional processing to become mature mRNAs. The typical progression of pre-mRNA maturation involves 5′-end capping, splicing, and 3′-end cleavage and polyadenylation. The integrity and precision of each of these steps are critical for generating stable, functional mRNAs. Moreover, recent studies have demonstrated the importance of alternative splicing, alternative polyadenylation (APA), and RNA editing in producing an incredibly diverse, often cell-specific mRNA library that contributes to the biological complexity of higher eukaryotes. <br> 5′-end capping occurs very early during Pol II transcription, typically after the synthesis of ∼20 nucleotides of the pre-mRNA. Capping has been linked to splicing and 3′-end processing of the pre-mRNA, and the export of the mature mRNA. In addition, the 5′-end cap is directly recognized by the eukaryotic translation initiation factor eIF-4E, which is essential for mRNA translation by the ribosome.</p>

<img src="img/transcription.png" alt="Transcription"/>

<p>During transcription, the genetic information stored in DNA is copied through synthesis of RNA. RNA synthesis is accomplished using RNA polymerase enzyme, with DNA as a template and a ribonucleoside monophospate ATP, CTP, GTP and UTP as RNA precursors. The RNA strand formed with the direction of synthesis being 5' → 3', is called <b>messager RNA (mRNA)</b>; Chain elogation occurs by adding the appropriate ribonucleoside monophospate residue to the free 3' hydroxyl group at the 3' end end of the growing RNA chain. As the RNA transcript is complementary to this <b>template strand</b>, the transcript has the same 5' → 3' direction and base sequence (except that U replace T) as the opposite, nontemplate strand of the double helix. For this reason the nontemplate strand is often called the <b>sense strand</b>, and the template stranf is often called the <b>antisense strand</b>.</p>

<img src="img/trascrizione.png" alt="trascrizione"/>

<h2>Maturation of pre-mRNA</h2>

<p>A typical eukaryotic protein-coding gene is transcribed into a precursor-mRNA (pre-mRNA). To produce a translatable mRNA, a pre-mRNA undergoes pre-mRNA processing that involves modifying each end of the molecule, and removing non-protein regions called <i>introns</i>. The mRNA exits the nucleus and is translated in the cytoplasm.</p>

<p>At the 5' end of a pre-mRNA is a modified GTP molecule called the <b>5'</b> cap. The 5' cap is added to the pre-mRNA by a <i>capping enzyme</i> soon after RNA polymerase II begins transcription and remain when pre-mRNA is processed to mRNA. THe cap is the site where a ribosome attaches to an mRNA at the start of translation, and it also protects the 5' end from RNA-digesting enzymes. <br> A eukaryotic protein-coding gene has no terminator sequence that signals RNA polymerase to stop transcription. Instead, near the 3' end of the gene is a sequence called a <b>polyadenylation signal</b> that is transcribed into the pre-mRNA. Proteins bind to this sequence, and cleave the pre-mRNA just downstream of that point. Then, the enzyme <b>poly(A) polymerase</b> adds a chain of 50 to 250 adenine nucleotides, one nucleotide at a time, to the 3' end of the pre-mRNA. The string of A nucleotides, called the <b>poly (A) tail</b>, enables the mRNA produced from the pre-mRNA to be translated efficiently, and protects it from attack by RNA-digesting enzymes in the cytoplasm. If the poly(A) tail of an mRNA is removed experimetally, the mRNA is degraded quickly inside cells.</p>

<p>The transcription unit of a typical eukaryotic protein-coding gene − the RNA-coding sequence− consists of <b>exons</b>, which are protein coding sequencing, interrupted by <b>introns</b>, which are non-protein-coding sequences. The majority of known eukaryotic genes contain at least one intron; some contain more than 60.</p>

<figure>
<img src="img/RNA_maturation.png" alt="Maturation RNA"/>
<figcaption>Relationship between a eukaryotic protein-coding gene, the pre-mRNA transcribed from it, and the mRNA processed from the pre-mRNA.</figcaption>
</figure>


<h3>mRNA splicing</h3>

<p>In the nucleus, a process called <b>mRNA splicing</b> removes introns from pre-mRNAs and splices exons together to produce a functional mRNA with the protein-coding sequence of the gene. mRNA splicing takes place in a <b>spliceosome</b>, a complex fromed between the pre-mRNA and a handful of <b>small nuclear ribonucleoprotein particles</b> (<b>snRNPs</b>, pronounced "snurps")- A ribonucleoprotein particle is a complex of RNA and protein. Each of the snRNPs involved in mRNA splicing consists of a short <i>small nuclear</i> RNA (SnRNA), a type of noncoding RNA, bound to several proteins. Key to mRNA splicing is base-pairing of particular snRNAs with specific sequences at the 5' and 3' ends of an intron that define the junctions between the intron and the adjacent exons. The spliceosome cleaves the pre-mRNA to release the intron, and joins the flanking exons. <br> The cutting and splicing are so exact that not a single base of an intron is retained in the finished mRNA, nor is a single base removed from an exon. Without this precision, removing intronds would change the reading frame of the coding portion of the mRNA, producing gibberish from the point of a mistake onward. Once the finished mRNA is generated it can be exported from the nucleus to be translated in the cytoplasm. Regulatory mechanism prevent pre-mRNs from being exported.</p>

<figure>
<img src="img/mRNA_splicing.png" alt="mRNA_splicing"/>
<figcaption>mRNA splicing − the removal from pre-mRNA of introns and joining of exons in the spliceosome.</figcaption>
</figure>

<h2>RNA Polymerase Resembles Other Polymerases</h2>

<p>The <i>E. coli</i> RNAP <b>holoenzyme</b> is an ∼ 449-kD protein with subunit composition α<sub>2</sub>ββ′ωσ (Table 26-2). Once RNA synthesis has been initiated, however, the σ subunit (also called the σ factor) dissociates from the <b>core enzyme</b>, α<sub>2</sub>ββ′ω, which carries out the actual polymerization process. RNAP is large enough to be clearly visible in electron micrographs.</p>

<p>The X-ray structure of the related RNAP core enzymes from Thermus aquaticus (Taq) and Thermus thermophilus (Tth) were independently determined by Seth Darst and Dmitry Vassylyev. The structure of the Tth core enzyme (which closely resembles that of the Taq enzyme) in complex with DNA and RNA has the overall shape of a crab claw whose two pincers are formed by the β and β′ subunits (Fig. 26-2). In this so-called open complex, the ∼ 27-Å-high space between the pincers, the main channel, is occupied by the dsDNA. The template DNA continues to the active site at the end of the main channel where it base-pairs with the incoming NTP (not present in the structure) at the so-called <i>i</i> + 1 site near a bound Mg<sup>2+</sup> ion. The 3′ end of the newly synthesized RNA forms a 9-bp hybrid helix with the 5′ end of the template DNA strand and then exits the protein through a channel between the β and β′ subunits, the RNA exit channel, in which it adopts a conformation similar to that of a single strand within an RNA double helix. However, the paths taken by the template and nontemplate DNA strands to rejoin at the end of the so-called <b>transcription bubble</b> are unclear.<br> In the X-ray structure of the Tth holoenzyme in complex with dsDNA containing a promoter (a sequence that specifi es the transcription initiation site; Section 26-1B), determined by Richard Ebright, the σ subunit extends across the core enzyme, which causes its pincers to come together so as to narrow the channel between them by ∼10 Å, thereby forming the so-called closed complex (Fig. 26-3). In this structure, the σ subunit has opened the transcription bubble by specifi cally binding the nontemplate strand’s –10 promoter element. The outer surface of the holoenzyme is almost uniformly negatively charged, whereas those surfaces that interact with nucleic acids, particularly the inner walls of the main channel, are positively charged.</p>

<h2>Only One Strand of DNA Is Copied</h2>

<p>RNA synthesis is normally initiated only at specific sites on the DNA template. In contrast to replication, which requires that both strands of the chromosome be entirely copied, the regulated expres-
sion of genetic information involves the copying of much smaller, single-strand portions of the genome. The DNA strand that serves as a template during transcription is known as the antisense or noncoding strand because its sequence is complementary to that of the RNA. The other DNA strand, the nontemplate strand, which has the same sequence as the transcribed RNA (except for the replacement of U with T), is known as the sense or coding strand (Fig. 26-4). The two strands of DNA in an organism’s chromosome can therefore contain different sets of genes.</p>

<h2>Transcription Is Initiated at a Promoter</h2>

<p>How does RNAP recognize the correct DNA strand and initiate RNA synthesis at the beginning of a gene (or operon)? RNAP binds to its initiation sites through base sequences known as <b>promoters</b> that are recognized by the corresponding <i>σ</i> factor. The existence of promoters was first revealed through mutations that enhanced or diminished the transcription rates of certain genes. Promoters consist
of ∼ 40-bp sequences that are located on the 5′ side of the transcription start site. By convention, the sequence of this DNA is represented by its sense (nontemplate) strand so that it will have the same sequence and directionality as the transcribed RNA. A base pair in a promoter region is assigned a negative or positive number that indicates its position, upstream or downstream in the direction
of RNAP travel, from the first nucleotide that is transcribed to RNA; this start site is +1 and there is no 0. Because RNA is synthesized in the 5′ → 3′ direction (see below), the promoter is said to lie upstream of the RNA’s starting nucleotide. <br> The holoenzyme forms tight complexes with promoters. This tight binding can be demonstrated by showing that the holoenzyme protects the bound DNA segments from digestion in vitro by the endonuclease DNase I. Sequence determinations of the protected regions from numerous E. coli genes have identified the “consensus” sequence of E. coli promoters (Fig. 26-6). Their most conserved sequence is a hexamer centered at about the −10 position (alternatively called the Pribnow box after David Pribnow, who described it in 1975). It has a consensus sequence of TATAAT in which the leading TA and the final T are highly conserved. Upstream sequences around position –35 also have a region of sequence similarity, TTGACA. The initiating (+1) nucleotide, which is nearly always A or G, is centered in a poorly conserved CAT or CGT sequence. Most promoter sequences vary considerably from the consensus sequence (Fig. 26-6). Nevertheless, a mutation in one of the partially conserved regions can greatly increase or decrease a promoter’s initiation efficiency. This is because the rates at which E. coli genes are transcribed vary directly with the rates at which their promoters
form stable initiation complexes with the RNAP holoenzyme.</p>

<table>
<tr><th>Operon</th> <th>−35 region</th> <th>10 region<br>(Pribnow box)</th> <th>Initiation<br>site (+1)</th> </tr>

<tr><td>lac</td> <td>ACCCCAGGC<b>TTTACA</b>CTTTATGCTTCCGGCTCG<b>TATGTT</b>GTGTG<b>GA</b>ATTGTGAGCGG</td> </tr>

<tr><td>lac I</td> <td>CCATCGAAT<b>GGCGCA</b>AAACCTTTCGCGGTATGG<b>CATGAT</b>AGCGCCC<b>G</b>GAAGAGAGTC</td></tr>

</table>

<h2>Operons</h2>

<p>Most protein-coding genes (called <b>structural genes</b>) in eukaryotes are transcribed individually. In prokaryotic genomes, however, genes are frequently arranged in tandem along a single DNA strand so that they can be transcribed together. These genetic units, called operons, typically contain genes with related functions. For example, the three different rRNA genes of <i>E. coli</i> occur in single <b>operons</b> (Section 26-3C). The <i>E. coli</i> <b>lac operon</b> (<i>lac</i> from lactose), whose expression is described in detail in Section 28-2A, contains three genes encoding proteins involved in lactose metabolism as well as sequences that control their transcription (Fig. 26-5). Other operons contain genes encoding proteins required for biosynthetic pathways, for example, the trp operon, whose six gene products (proteins are often referred to as gene products) catalyze tryptophan synthesis. An operon is transcribed as a single unit, giving rise to a polycistronic mRNA that directs the more-or-less simultaneous synthesis of each of the encoded polypeptides (the term cistron is a somewhat archaic synonym for gene). In contrast, eukaryotic structural genes, which are not part of operons, give rise to <b>monocistronic mRNAs</b>.</p>

<figure>
<img src="img/lac_operon.png" alt="lac operon"/>
<figcaption><b>The <i>E. coli</i> lac operon</b>. This DNA includes genes encoding the proteins mediating lactose metabolism and the genetic sites that control their expression. The Z, Y, and A genes, respectively, specify the proteins β-galactosidase (Box 8-1), galactoside permease (Section 10-3D), and thiogalactoside transacetylase. The closely linked regulatory gene, I, which is not part of the lac operon, encodes a repressor that inhibits transcription of the lac operon</figcaption>
</figure>

<table>

<caption>Table 1: Overview of Differences Between Prokaryotic and Eukaryotic Gene Expression and Regulation</caption>

<tr> <th></th> <th>Prokaryotes</th> <th>Eukaryotes</th> 

<tr> <th>Structure of genome</th><td>Single, generally circular genome sometimes accompanied by smaller pieces of accessory DNA, like plasmids</td> <td>Genome found in chromosomes; nucleosome structure limits DNA accessibility</td></tr>

<tr><th>Size of genome</th> <td>Relatively small</td> <td>Relatively large</td></tr>

<tr><th>Location of gene transcription and translation</th> <td>Coupled; no nucleoid envelope barrier because of prokaryotic cell structure</td> <td>Nuclear transcription and cytoplasmic translation</td></tr>

<tr><th>Gene clustering</th> <td>Operons where genes with similar function are grouped together</td> <td>Operons generally not found in eukaryotes; each gene has its own promoter element and enhancer element(s)</td></tr>

<tr><th>Default state of transcription</th> <td>On</td> <td>Off</td></tr>

<tr><th>DNA structure</th> <td>Highly supercoiled DNA with some associated proteins</td> <td>Highly supercoiled chromatin associated with histones in nucleosomes</td></tr>

</table>
 
<section aria-label="End">
<a href="dlconfiguration.html">&#171;D- and L- configuration</a>
<a href="index.html">Index</a>
<a href="traduzione.html">Traduzione&#187;</a>
</section>

</div>

</body>
</html>
